copier/main.py

```diff
@@ -218,6 +218,8 @@ def _execute_tasks(self, tasks: Sequence[Task]) -> None:
         Arguments:
             tasks: The list of tasks to run.
         """
+        if self.pretend:
+            return
```

Comment:
I think it should still print something, saying what you are pretending to do.

Reply:
Good point. :+1: I've changed it accordingly.